ckan/lib/plugins.py

```diff
@@ -524,3 +524,31 @@ def activity_template(self):
         return 'organization/activity_stream.html'
 
 _default_organization_plugin = DefaultOrganizationForm()
+
+
+class DefaultTranslation(object):
+    def directory(self):
+        '''Change the directory of the *.mo translation files
+
+        The default implementation assumes the plugin is
+        ckanext/myplugin/plugin.py and the translations are stored in
+        i18n/
+        '''
+        import os
+        return os.path.abspath(os.path.join(os.path.dirname(__file__),
```

Comment:
fair enough, I've never really messed with the manifest stuff, but it sounds like it'll have to go in that location, it'll fix my test problem as well.